qiskit/schemas/qobj_schema.json

```diff
@@ -871,7 +878,14 @@
                     "description": "Pulse instructions implementing the given gate.",
                     "type": "array",
                     "items": {
-                        "$ref": "#/definitions/qobjinstructions"
+                        "oneOf": [
+                            {
+                                "$ref": "#/definitions/parameter"
+                            },
+                            {
+                                "$ref": "#/definitions/qobjinstructions"
```

Comment:
The instructions and params will contain "parameter"s too

```
calibration.gates = [
    {'name': 'rx', 'qubits': [TODO], 'params': [TODO], 'instructions': [{'name': 'setf', 'frequency': TODO}]`}
]
```

The TODOs are what you have to decide on. I mentioned looking at the `default_pulse_configuration` because I think it has chosen one way of doing this:

```
calibration.gates = [
    {'name': 'rx', 'qubits': ['P0', 'P1'], 'params': ['0.5*P2'], 'instructions': [{'name': 'setf', 'frequency': 'P2'}]`}
]
```

Reply:
I'm not sure if I understood this comment. Currently, we have,
```python
circuit.calibrations = 
    {'gate_name': {(qubits, params): schedule}}
```
Is the ```calibrations.gates``` WIP or should I add it in this PR?